src/test/clojure/xtdb/expression_test.clj

```diff
@@ -1573,6 +1573,17 @@
                                   #(->dur-vec "x" TimeUnit/SECOND 10)
                                   #(->bigint-vec "y" 3))))))))
 
+(t/deftest cast-duration-test
```

Comment:
I don't think this will work with anything other than `:milli`?

Reply:
hopefully now after checking and using desired resolution this should work

Reply:
much better, thanks :relaxed: :pray: